substrate/client/rpc-servers/src/lib.rs

```diff
@@ -43,7 +42,7 @@ pub use jsonrpsee::{
 	core::id_providers::{RandomIntegerIdProvider, RandomStringIdProvider},
 	server::{middleware::rpc::RpcServiceBuilder, BatchRequestConfig},
 };
-pub use middleware::{Metrics, MiddlewareLayer, RpcMetrics};
+pub use middleware::{Metrics, MiddlewareLayer, NodeHealthProxyLayer, RpcMetrics};
```

Comment:
why is it useful to export the middleware implementation?
it's not possible to opt in/out from it anyway?

Reply:
I need to reimpl these rpc-servers.

Reply:
I don't follow, you want to write your own `start server` or an API to inject your own http middleware? 
Anyway, doesn't hurt to re-export it from my-side

Reply:
It would be better to write an issue which what functionality you want.

We could expose APIs for folks to inject their own middleware both http and rpc... but it's a little tricky to get the traits bounds satisfied properly.